src/components/HomeInfo.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Button } from '../atoms/Button';
+import { Avatar } from '../atoms/Avatar';
+
+class HomeInfo extends React.Component {
+    render() {
+        return (
+            <div className="container-homeInfo">
+                <h3>subtitle</h3>
+                <h1>title to tilorem</h1>
+                <div className="container-messager">
+                    <Avatar />
+                    <span className="message">"It is a long established fact that a reader will be distracted by the readable content of a page."</span>
+                </div>
+                <Button />
```

Comment:
After you fix the Button component, you can pass here its children to be its text:

```jsx
<Button>Some text you want</Button>
```